packages/shared/src/modules/reviews/libs/types/review-get-all-response-dto.type.ts

```diff
@@ -0,0 +1,15 @@
+type ReviewGetAllResponseDto = {
```

Comment:
Get all response dto is array. To define a type for item in it, we add item suffix
```suggestion
type ReviewGetAllItemResponseDto = {
```